### ISSUE TITLE
indexOf on IE 11 (emulated IE8)

### ISSUE THREAD

**Issue by toto-le-motard** [jackocnr/intl-tel-input#547]
I was using v 10.0.0 but I had the same issue as cook Island for a french phone number. So I update the version to 10.0.8 but I have an unexpected error on the line 654 : 
`var countryCodes = this.countryCodes[this._getNumeric(dialCode)], alreadySelected = this.selectedCountryData && **countryCodes.indexOf(this.selectedCountryData.iso2)** > -1`

Here is my values infos (in debug) :
 - dialCode="+33"
 - countryCodes = [fr]
 - this.selectedCountryData.iso2 = "fr"

### Expected behaviour
The value of iso2 must be found in the map

### Actual behaviour
The indexOf method generate an unexpected error with IE11 (emulating IE8 in my case)

## Analyses
I think this bug appair with the 22 december commit. I tried with the code before this commit in debug and it's works:
`$.inArray(this.selectedCountryData.iso2, countryCodes) != -1`

**Comment by jackocnr** [comment]
Hmm yeah it seems indexOf is only supported by IE9+. Thanks for reporting this. While we're supporting IE8 we should switch back to $.inArray. I just had a look and we actually have 3 occurrences of indexOf now. Unfortunately I don't have time to implement this myself right now, but would be open to a pull request!

**Comment by caseyjhol** [comment]
I recommend simply using a polyfill for IE8, considering the performance improvements of using the native indexOf, and the rarity of IE8 usage (fewer than 1%).

From: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/indexOf

```js
if (!Array.prototype.indexOf) {
  Array.prototype.indexOf = function(searchElement, fromIndex) {

    var k;

    if (this == null) {
      throw new TypeError('"this" is null or not defined');
    }

    var o = Object(this);

    var len = o.length >>> 0;

    if (len === 0) {
      return -1;
    }

    var n = fromIndex | 0;

    if (n >= len) {
      return -1;
    }

    k = Math.max(n >= 0 ? n : len - Math.abs(n), 0);

    while (k < len) {
      if (k in o && o[k] === searchElement) {
        return k;
      }
      k++;
    }
    return -1;
  };
}

**Comment by jackocnr** [comment]
@caseyjhol thanks for your input, but actually jQuery defers to native `indexOf` if it's available.

Fixed this issue in v10.0.10.

Ps. Soon I will be releasing v11.0.0 which will drop support for IE8 (see readme for reasoning).
